packages/proofs/semaphore-proof/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@cryptkeeperzk/semaphore-proof",
+  "version": "0.1.0-beta.1",
+  "description": "A forked Semaphore proof library to generate and verify Semaphore proofs.",
+  "license": "MIT",
+  "main": "dist/index.node.js",
+  "exports": {
+    "import": "./dist/index.mjs",
+    "require": "./dist/index.node.js"
+  },
+  "types": "dist/types/index.d.ts",
+  "files": [
+    "dist/",
+    "src/",
+    "LICENSE",
+    "README.md"
+  ],
+  "repository": "https://github.com/semaphore-protocol/semaphore",
+  "homepage": "https://github.com/semaphore-protocol/semaphore/tree/main/packages/proof",
+  "bugs": {
+    "url": "https://github.com/semaphore-protocol/semaphore.git/issues"
+  },
+  "scripts": {
+    "build:watch": "rollup -c rollup.config.ts -w --configPlugin typescript",
+    "build": "rm -rf dist && tsc -p build.tsconfig.json",
+    "prepublishOnly": "yarn build",
```

Comment:
we use pnpm let's add it here

Reply:
Actually we dont need this script in this package.